release_notes/ocp-4-17-release-notes.adoc

```diff
@@ -441,6 +441,13 @@ Starting in {product-title} 4.14, Extended Update Support (EUS) is extended to t
 [id="ocp-4-17-registry"]
 === Registry
 
+[id="ocp-4-17-chunksizemib-configuration-parameter"]
+==== New chunkSizeMiB configuration parameter for AWS S3 registry storage
```

Comment:
chunkSizeMiB is not only for aws s3, it could be used for the s3 api compatible storage